### PR TITLE
Add William Oliveira

### PR DESCRIPTION
**Pull request by woliveiras**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -104,6 +104,7 @@
 - [Ariya Hidayat](https://github.com/ariya/ama) - VP of Engineering [Shape Security](https://shapesecurity.com/), Author of [Esprima](https://github.com/jquery/esprima) and [PhantomJS](http://phantomjs.org).
 - [Rob Thomas](https://github.com/xrobau/ama) - VoIP Developer, IPv6 and Security Evangelist, DevOps proponent. Currently at [Sangoma](https://www.sangoma.com).
 - [Amit Merchant](https://github.com/amitmerchant1990/ama) - Full-stack developer working with JavaScript, PHP and Java.
+- [William Oliveira](https://github.com/woliveiras/ama) - Front End Developer (Brazilian) working with JavaScript, Angular, Stylus, Gulp, Git, etc. Look more about me on [my blog](http://woliveiras.com.br)
```
Comment (sindresorhus) [inline readme.md]: > Look more about me on my blog

Is not a valid sentence. Also needs to end in a dot.

**Comment by woliveiras** [comment]
Fixed @sindresorhus :smile:

**Comment by sindresorhus** [comment]
> Look more about me on my blog.

Is not fixed. It's not a valid sentence.

You could say:

> Read more about me on my blog.

**Comment by woliveiras** [comment]
Sorry @sindresorhus. Updated.

**Comment by sindresorhus** [comment]
Welcome! :)

Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). Especially the ones closer to the bottom of the list. I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=

**Comment by woliveiras** [comment]
Thanks! :D

**Pull request closed without merging**